src/components/About/About.tsx

```diff
@@ -38,10 +38,26 @@ export function About() {
           <h3 className={styles.h3New}>Triathalon Coaching</h3>
           <h2 className={styles.h2}>Sam Pausha</h2>
         </div>
-        <p className={styles.aboutParagraph}>{ABOUT_BIO}</p>
+        <p className={styles.aboutParagraph}>
+          {ABOUT_BIO.split("\n").map((line, index) => {
```

Comment:
you can just have it like it was before but add white-space: pre in css I believe. Some value of white-space respects \n like you want